tool/rbinstall.rb

```diff
@@ -376,6 +376,11 @@ def CONFIG.[](name, mandatory = false)
   if rubyw_install_name and !rubyw_install_name.empty?
     install rubyw_install_name+exeext, bindir, :mode => $prog_mode, :strip => $strip
   end
+  # emcc produces ruby and ruby.wasm, the first is a JavaScript file of runtime support
+  # to load and execute the second .wasm file. Both are required to execute ruby
+  if RUBY_PLATFORM =~ /emscripten/ and ruby_install_name+".wasm"
```

Comment:
The RHS will be always truthy.
You may want to use `File.exist?`?

Reply:
Ah, you are right.